sdk/examples/devmode_rust/src/engine.rs

```diff
@@ -214,7 +214,7 @@ impl Engine for DevmodeEngine {
         updates: Receiver<Update>,
         service: Box<Service>,
         startup_state: StartupState,
-    ) {
+    ) -> Result<(), Error>{
```

Comment:
Should probably be `Result<(), Error> {` instead of `Result<(), Error>{`

Reply:
Good catch, updated 👍